app/src/features/datasets/DatasetsRouter.tsx

```diff
@@ -11,10 +10,6 @@ import DatasetsPage from './DatasetsPage';
 const datasetsRouter: React.FC<React.PropsWithChildren> = () => {
   return (
     <Switch>
-      <AppRoute exact path="/datasets">
-        <DatasetsPage />
-      </AppRoute>
-
       <AppRoute exact path="/datasets/:id/details">
```

Comment:
(Optional nice to have) you could add a redirect line above this route, so that if someone navigates to `/datasets/:id` it redirects them to `/datasets/:id/details` (if that is the default page for that route).

Example is here, for the project details route: https://github.com/bcgov/biohubbc/blob/dev/app/src/features/projects/ProjectsRouter.tsx

Reply:
That being said, not sure why they would navigate to just `/datasets/:id` in the first place